src/App.spec.ts

```diff
@@ -761,30 +814,6 @@ describe('App', () => {
           // Assert
           assert.equal(assertionAggregator.callCount, dummyReceiverEvents.length);
         });
-
-        it('should handle failures through the App\'s global error handler', async () => {
```

Comment:
I believe this is failing b/c the global handler doesn't know how to handle thrown errors (yet). I'm adding this test back in a later pr. Will link it when it's up.

Reply:
Added back https://github.com/slackapi/bolt/pull/380/commits/269f462e57c069e1319cb78d6c610a44c95591cc#diff-a7a8d4758cfeb70999a809869b45fca0R976